recipes/TIMIT/ASR/transducer/hparams/train.yaml

```diff
@@ -51,7 +55,7 @@ dnn_neurons: 512
 dec_neurons: 128
 
 # Outputs
-output_neurons: 40
+output_neurons: 42
```

Comment:
why 42 ?

Reply:
In my recipe tests, I had errors because the units (according the label_dictionary) are actually 42. I will check better later